target/linux/ath79/dts/qca9531_8dev_lima.dts

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+        compatible = "8dev,lima", "qca,qca9531";
+        model = "8devices Lima";
+
+        chosen {
+		bootargs="console=ttyS0,115200n81";
```

Comment:
In case you mean n8 instead of n81, the chosen node can be dropped entirely due to a recent patch on master.

Reply:
Fixed.